src/golang/cmd/server/handler/connect_integration.go

```diff
@@ -320,6 +320,12 @@ func ValidateConfig(
 		return validateLambdaConfig(ctx, config)
 	}
 
+	if service == integration.Databricks {
```

Comment:
In https://www.notion.so/aqueducthq/In-memory-struct-interface-requirement-collections-brainstorming-be252894dcf2402ba27db28c980c79e6 I had one section to move these `validate` to interface and implementation pattern so that we can avoid these large section of `if` statements